content/useStyles: a hook to manage your styles - Agustín Prieto - EN.md

```diff
@@ -0,0 +1,64 @@
+![Banner React Native](images/useStylesBanner.jpg)
+
+# useStyles: a hook to manage styles in React Native
+
+¿Does anyone know how to properly manage styles in React Native? I don't.
+
+I have been working with this framework for a couple of years now and couldn't get a concise way of using styles. This last year I've tried some libraries with novel concepts as Styled Components, but I couldn't get used to them. As a result of this dissatisfaction with the current alternatives, useStyles was born.
+
+![One hook to rule them all](images/useStylesMeme.jpg)
+
+## What is useStyles and how can it help me?
+
+Organizing styles in React Native can be tricky, a headache if you don't apply good programming practices. useStyles comes with the idea of ​​alleviating this burden by providing the tools so that you can generate applications where the styles have their own place and can reside harmoniously with the components.
```

Comment:
```suggestion
Organizing styles in React Native can be tricky, a headache if you don't apply good programming practices. useStyles comes with the purpose of ​​alleviating this burden by providing the tools so that you can generate applications where the styles have their own place and can reside harmoniously with the components.
```